README.md

```diff
@@ -343,6 +344,57 @@ helm install valheim-server valheim-k8s/valheim-k8s # see repo for full config
 CDK Project for spinning up a Valheim game server on AWS Using ECS Fargate and Amazon EFS is available here:
 [https://github.com/rileydakota/valheim-ecs-fargate-cdk](https://github.com/rileydakota/valheim-ecs-fargate-cdk)
 
+## Deploying to Nomad
+Copy and Paste the following to `valheim.nomad`:
```

Comment:
Just add `valheim.nomad` to the repo and change instructions to something like:
```
$ sudo mkdir -p /var/lib/valheim/{config,data}
$ sudo curl -o /var/lib/valheim/valheim.nomad https://raw.githubusercontent.com/lloesche/valheim-server-docker/main/valheim.nomad
$ sudo nomad run job /var/lib/valheim/valheim.nomad
```
or whatever the specifics are.